src/PrestaShopBundle/Bridge/AdminController/ControllerConfiguration.php

```diff
@@ -193,7 +193,23 @@ class ControllerConfiguration
     public $modals = [];
 
     /**
+     * This parameter is needed by legacy helper shop, we can't remove it.
+     *
      * @var int
      */
-    public $multishopContext = Shop::CONTEXT_ALL | Shop::CONTEXT_GROUP | Shop::CONTEXT_SHOP;
+    public $multishop_context = Shop::CONTEXT_ALL | Shop::CONTEXT_GROUP | Shop::CONTEXT_SHOP;
```

Comment:
Actually, we could keep this but it can be done in a refacto. The idea would be to transform this DTO object by relying on protected fields and public getters/setters Thus adding stricter control on the type of the fields and even making some of those fields immutable: controllerName or controllerNameLegacy for example shouldn't be modified, they could be provided in the constructor and never be modified then.

Then the bridge magic getter could handle the "conversion" process between the old format via a simple configuration, naive idea would be like:

```php
    const MAPPING = [
        'controller_name' => 'getControllerName',
        'php_self' => 'getControllerName',
        'token' => 'getToken',
    ]
```

Most of those could actually be handled relying on a `PropertyAccessor` Symfony component, but some (like php_self) may need some extra conversion.

But anyway it's not the scope of this PR.